src/main/java/de/teamlapen/vampirism/entity/converted/EntityConvertedHorse.java

```diff
@@ -0,0 +1,105 @@
+package de.teamlapen.vampirism.entity.converted;
+
+import de.teamlapen.vampirism.api.EnumStrength;
+import de.teamlapen.vampirism.api.VReference;
+import de.teamlapen.vampirism.api.VampirismAPI;
+import de.teamlapen.vampirism.api.entity.convertible.IConvertedCreature;
+import de.teamlapen.vampirism.api.entity.convertible.IConvertingHandler;
+import de.teamlapen.vampirism.util.Helper;
+
+import net.minecraft.entity.EntityCreature;
+import net.minecraft.entity.EntityLivingBase;
+import net.minecraft.entity.ai.EntityAIAvoidEntity;
+import net.minecraft.entity.ai.EntityAIHurtByTarget;
+import net.minecraft.entity.ai.EntityAINearestAttackableTarget;
+import net.minecraft.entity.ai.EntityAIRestrictSun;
+import net.minecraft.entity.passive.EntityHorse;
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.init.MobEffects;
+import net.minecraft.nbt.NBTTagCompound;
+import net.minecraft.potion.PotionEffect;
+import net.minecraft.util.math.MathHelper;
+import net.minecraft.world.World;
+
+public class EntityConvertedHorse extends EntityHorse implements IConvertedCreature<EntityHorse> {
+
+    private EnumStrength garlicCache = EnumStrength.NONE;
+    private int bloodTimer = 0;
+
+    public EntityConvertedHorse(World world) {
+        super(world);
+    }
+
+    @Override
+    public boolean doesResistGarlic(EnumStrength strength) {
+        return !strength.isStrongerThan(EnumStrength.NONE);
+    }
+
+    @Override
+    public void drinkBlood(int amt, float saturationMod, boolean useRemaining) {
+        this.addPotionEffect(new PotionEffect(MobEffects.REGENERATION, amt * 20));
+        bloodTimer = -1200 - rand.nextInt(1200);
+    }
+
+    @Override
+    public EnumStrength isGettingGarlicDamage(boolean forceRefresh) {
+        if (forceRefresh) {
+            garlicCache = Helper.getGarlicStrength(this);
+        }
+        return garlicCache;
+    }
+
+    @Override
+    public boolean isGettingSundamage(boolean forceRefresh) {
+        // TODO Auto-generated method stub
+        return false;
+    }
+
+    @Override
+    public boolean isIgnoringSundamage() {
+        // TODO Auto-generated method stub
+        return false;
+    }
+
+    @Override
+    public boolean useBlood(int amt, boolean allowPartial) {
+        this.addPotionEffect(new PotionEffect(MobEffects.WEAKNESS, amt * 20));
+        bloodTimer = 0;
+        return true;
+    }
+
+    @Override
+    public boolean wantsBlood() {
+        return bloodTimer > 0;
```

Comment:
But blood timer is never increased.

Also, since they will never suck blood, we probably do not need this at all